src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx

```diff
@@ -0,0 +1,80 @@
+
+### Lifecycle management of sample applications
+
+The [Meshery Adapter for Consul](https://meshery.layer5.io/docs/service-meshes/adapters/consul) comes bundled with a handful of sample applications for evaluating, exploring, and learning how to operate Consul service mesh. Many operators are new to the ongoing administrative tasks of running a healthy and optimized Consul service mesh deployment. In advance of their production deployments, operators may utilize Meshery to quickly deploy Consul with sample applications to gain familiarity with the many features of Consul.
+
+Demonstrated at DockerCon 2020, is the “[Image Hub](https://github.com/layer5io/image-hub)”, a sample application built to allow users to explore Consul’s feature set, and specifically, an experimental area of Consul’s data plane: Envoy’s impending support for WebAssembly.
+
+### Performance management of Consul and it’s workloads
+
+In order to assess the overall performance of the service mesh, and the overhead of individual, fine-grained traffic control mechanisms defined in Consul’s control plane and enforced through Consul’s intelligent data plane, Meshery provides users with statistical analysis of the responsiveness of their services and performance of the service mesh. As highlighted by Docker Captain, [Luc Juggery](https://twitter.com/lucjuggery), performance is an ongoing concern:
+
+<div style="margin:20px;">Running a performance test is not a one shot thing. Tests should be run on a regular basis to (re)establish baselines and evaluate configuration changes:
+<ul>
+<li>- for each new release of the chosen service mesh</li>
+<li>- for each change of the configuration of your service mesh</li>
+<li>- for each new release of the application"</li>
+</ul>
+</div>
+Meshery’s ability to connect to Prometheus instances to retrieve and account for cluster and application-level metrics is popularly used during Meshery’s service mesh performance tests. Likewise, Meshery’s ability to connect to and import existing dashboards, panels, and charts from Grafana is instrumental in allowing users to retain their existing investment in dashboards and metrics they have curated over time.
+
+## Meshery Announces Experimental Support for WebAssembly using Consul
+
+Today, at [DockerCon 2020](https://docker.events.cube365.net/docker/dockercon/content/Videos/63TCCNpzDC7Xxnm8b), we demonstrate technology leadership in advanced data plane engineering for near-native performance of fine-grained traffic control facilitated by Meshery and Consul with the use of Envoy and WebAssembly. 
+
+<a href="./2020-05-28-layer5-hashicorp-launch-service-mesh-partnership/layer5-image-hub-on-hashicorp-consul.png"><img src="layer5-image-hub-on-hashicorp-consul.png" width="100%" target=_blank /></a>
```

Comment:
Sorry for the trouble @iambami. Here the `target=_blank` is not needed. You can remove that. 
@leecalcote I think the `a` tag wrapping the image is not necessary here since we use the `gatsby-image` plugin to dynamically load the image and it is very tough to link the image with `a` tag. Your thoughts? @leecalcote